testutils/generators.go

```diff
@@ -340,19 +338,27 @@ var mutex = &sync.Mutex{}
 func GenerateUniqueJetID() insolar.JetID {
 	for {
 		jetID := gen.JetID()
-		id := binary.BigEndian.Uint64(jetID.Prefix())
-		if id == 0 {
+		if !isUniqueJetID(jetID) {
 			continue
 		}
-		mutex.Lock()
-		_, hasKey := uniqueJetID[id]
-		if !hasKey {
-			uniqueJetID[id] = true
-			mutex.Unlock()
-			return jetID
-		}
+		return jetID
+	}
+}
+
+func isUniqueJetID(jetID insolar.JetID) bool {
```

Comment:
isUniqueJetID means check does this value is unique or not. that's a wrong way to do that here.